stories/form.fmf

```diff
@@ -0,0 +1,16 @@
+/machine-readable:
+    story:
+        As a Testing Farm I want to get a machine readable output from the service.
+
+    description: |
+        Testing Farm needs to be able to get a machine readable output from the service, various formats could be
+        supported, such as JSON, YAML, XML, etc.
+
+/human-readable:
+    story:
+        As a user I want to get a human readable output from the service.
+
+    description: |
+        Users need to get result after clicking on the link in JIRA in human-readable form, such as simple HTML page.
+        Can be formatted, would be provided by a web app that communicates with the service.
```

Comment:
What about something like this?

    After clicking on a link user will be presented a nicely
    formatted web page with overview of the linked test, plan or
    both test and plan. Among the information displayed there will
    be:

    * summary
    * description
    * ...